lib/venue.rb

```diff
@@ -1,2 +1,35 @@
 class Venue
+  attr_reader :name, :capacity, :patrons, :yelled_at
+  def initialize(name, capacity)
+  @name = name
+  @capacity = capacity
+  @patrons = []
+  @yelled_at = []
+  end
+
+  def add_patron(patron_name)
+    @patrons << patron_name
+  end
+
+  def yell_at_patrons
+    @patrons.each do |patron|
+      @yelled_at << patron.upcase
+    end
+    return yelled_at
+  end
+
+  def over_capacity?
+    return true if @patrons.count > @capacity
```

Comment:
You can get rid of everything before `@patrons.count > @capacity` as well as the `false`. By writing this:
```
@patrons.count > @capacity
```
it will return a boolean